### PR TITLE
Document cnf tests limitations.

### PR DESCRIPTION
**Pull request by fedepaol**

### CODE REVIEW

File: cnf-tests/README.md [hunk 1]
```diff
@@ -288,6 +290,12 @@ The nodes on which the tests will be executed can limited by means of specifying
   docker run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig -e NODES_SELECTOR=node-role.kubernetes.io/worker-cnf quay.io/openshift-kni/cnf-tests /usr/bin/test-run.sh
 ```
 
+### Clean Up - Known Limitations
+
+Temporary objects (such as namespaces, pods, ..) are deleted when the suite finishes.
+
+As a side effect, all the netork policies / ptp configurations starting with "test-" are deleted.
```
Comment (SchSeba) [inline cnf-tests/README.md:297]: `netork` -> `network`

Comment (fedepaol) [reply cnf-tests/README.md]: Done

**Pull request merged**